src/io/import_stl.cc

```diff
@@ -53,7 +53,7 @@ static void uint32_byte_swap(uint32_t& x) {
 
 static void read_stl_facet(std::ifstream& f, stl_facet& facet) {
   f.read((char *)facet.data8, STL_FACET_NUMBYTES);
-  if ((unsigned long int) f.gcount() < STL_FACET_NUMBYTES) {
+  if (static_cast<unsigned long int >(f.gcount()) < STL_FACET_NUMBYTES) {
```

Comment:
Just use `size_t` since that's what STL_FACET_NUMBYTES is defined as. Introducing a new type is just confusing.